fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/table/GenericTableLoader.java

```diff
@@ -38,15 +38,15 @@
 import javax.validation.constraints.NotNull;
 
 /**
- * Load the table configuration for any druid setup.
+ * Load the table configuration for any Druid setup.
  */
 public class GenericTableLoader extends BaseTableLoader {
-    private final Map<String, Set<Granularity>> dataSourceToValidGrains = new HashMap<>();
+    private final Map<String, Set<Granularity>> dataSourceToValidGrains;
     // Set up the metrics
-    private final Map<String, Set<FieldName>> dataSourceToDruidMetricNames = new HashMap<>();
-    private final Map<String, Set<ApiMetricName>> dataSourceToApiMetricNames = new HashMap<>();
+    private final Map<String, Set<FieldName>> dataSourceToDruidMetricNames;
+    private final Map<String, Set<ApiMetricName>> dataSourceToApiMetricNames;
     // Set up the table definitions
-    private final Map<String, Set<PhysicalTableDefinition>> dataSourceToTableDefinitions = new HashMap<>();
+    private final Map<String, Set<PhysicalTableDefinition>> dataSourceToTableDefinitions;
```

Comment:
Initializations are moved to constructor, because initialization in constructor seems to be more ubiquitous in Fili code base.